nova/compute/resource_tracker.py

```diff
@@ -1514,6 +1522,7 @@ def _get_usage_dict(self, object_or_dict, instance, **updates):
         :param instance: nova.objects.Instance for the related operation; this
                          is needed to determine if the instance is
                          volume-backed
+        :param bdms: Optional list of block-device-mappings for the instance
```

Comment:
This needs to be a `BlockDeviceMappingList` if I read `compute_utils.is_volume_backed_instance()` correctly. Because it calls `.root_bdm()` on this object.